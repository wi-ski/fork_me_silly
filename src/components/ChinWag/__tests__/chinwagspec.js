cd 
cd Proimport ChinWag from '../index'
'use strict';
jest.mock('react-dom');
jest.mock('react/lib/ReactDefaultInjection');
describe('ColorPicker', function() {

    beforeEach(() => {
        jest.resetModules()
        global.Renderer= require('react-test-renderer');
        global.React= require('react');
        global.TestUtils= require('react-addons-test-utils');
    });

    test('Link renders correctly', () => {
      const tree = Renderer.create(
        <ChinWag onKeyUp={function(){}} messages={[]} styles={{}} />
      ).toJSON();
      expect(tree).toMatchSnapshot();
    });
});