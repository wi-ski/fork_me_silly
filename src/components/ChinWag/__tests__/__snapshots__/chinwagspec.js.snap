exports[`ColorPicker Link renders correctly 1`] = `
<div
  className="chatContainerStyles"
  id="chinwag"
  style={undefined}>
  <ul
    className="nav nav-list messageListStyles"
    style={undefined} />
  <label>
    <input
      className="messageInputStyles"
      defaultValue="Hit me"
      onKeyUp={[Function]}
      style={undefined}
      type="text" />
  </label>
</div>
`;
